config/locales/de.yml

```diff
@@ -106,7 +106,7 @@ de:
       edit:
         title: "Benutzer bearbeiten"
         reset_password: "Passwort zurücksetzen"
-        warning: "Das zurücksetzen des Passworts eines Benutzers führt zum Verlust des Zugriff auf alle Private Teams und die löschung aller Teams, in denen er der einzige Member ist"
+        warning: "Das Zurücksetzen des Passworts eines Benutzers führt zum Verlust des Zugriff auf alle Private Teams und die Löschung aller Teams, in denen er der einzige Member ist"
```

Comment:
Das Zurücksetzen des Passworts eines Benutzers führt zum Verlust des Zugriffs auf alle privaten Teams sowie die Löschung aller Teams, in denen er der einzige Member ist.